editor/export/editor_export.cpp

```diff
@@ -110,8 +110,13 @@ void EditorExport::save_presets() {
 	save_timer->start();
 }
 
+void EditorExport::emit_presets_changed() {
+	emit_signal(_export_presets_changed);
+}
+
 void EditorExport::_bind_methods() {
-	ADD_SIGNAL(MethodInfo("export_presets_updated"));
+	ADD_SIGNAL(MethodInfo(_export_presets_updated));
+	ADD_SIGNAL(MethodInfo(_export_presets_changed));
```

Comment:
The names are quite similar, it's a bit hard to know at a glance which one means what.

Reply:
I wasn't sure how to name the new signal. It could be `export_preset_list_changed`, but that doesn't cover runnable change.

Reply:
Can we just use `_export_presets_updated` to signal both an addition to the list and an update to the `runnable` state?

I don't think we need to introduce a new signal.

Reply:
I didn't want to change what this signal does, but sure.

Reply:
Wait no, the reason why I didn't use the old signal is that it's emitted too often for my purposes. There is no reason to run the update code on unrelated changes.

Reply:
> Wait no, the reason why I didn't use the old signal is that it's emitted too often for my purposes. There is no reason to run the update code on unrelated changes.

I think it may be a compromise worth making; after all, it'll end up being called an order of magnitude less than it's being invoked in the current codebase.

Reply:
Or you could create a new signal specifically for the `runnable` update.